Cargo.lock

```diff
@@ -2121,15 +2686,14 @@ checksum = "e4a24736216ec316047a1fc4252e27dabb04218aa4a3f37c6e7ddbf1f9782b54"
 
 [[package]]
 name = "nix"
-version = "0.20.2"
+version = "0.20.0"
```

Comment:
the audit failed (among others) because of the downgrade of this version, because of [RUSTSEC-2021-0119](https://rustsec.org/advisories/RUSTSEC-2021-0119.html). 

Was there a specific reason to downgrade it? 

( I didn't dig into where we actually use the package, we could do that if you need this downgrad)

Reply:
I was just trying to find out what was going on here too 😁, the downgrade is caused by `nix 0.20.2` having a limited version range of `bitflags` that isn't compatible with what `git-config` needs:

```console
> cargo update -p nix --precise 0.20.2
error: failed to select a version for `bitflags`.
    ... required by package `nix v0.20.2`
    ... which satisfies dependency `nix = "^0.20.0"` of package `rustwide v0.15.0`
    ... which satisfies dependency `rustwide = "^0.15.0"` (locked to 0.15.0) of package `docs-rs v0.6.0 (/home/nemo157/sources/docs.rs)`
versions that meet the requirements `>=1.1.0, <1.3.0` are: 1.2.1, 1.2.0, 1.1.0

all possible versions conflict with previously selected packages.

  previously selected package `bitflags v1.3.2`
    ... which satisfies dependency `bitflags = "^1.3.2"` of package `git-config v0.7.1`
    ... which satisfies dependency `git-config = "^0.7.1"` (locked to 0.7.1) of package `git-repository v0.23.0`
    ... which satisfies dependency `git-repository = "^0.23.0"` (locked to 0.23.0) of package `crates-index-diff v11.1.1`
    ... which satisfies dependency `crates-index-diff = "^11.0.0"` (locked to 11.1.1) of package `docs-rs v0.6.0 (/home/nemo157/sources/docs.rs)`
```

We likely need to upgrade the `nix` dependency in `rustwide`, the newer versions of `nix` allow using newer versions of `bitflags`.

Reply:
Thanks so much for researching this! I wasn't even aware of the possibility of updating individual packages with `cargo update` - nice!

What I find interesting is that `git-config` forces as more recent version even though it most definitely doesn't use any special features of the more recent release. If it helps, I can also downgrade `bitflags` there to 1.2.1, even though waiting for the `rustwide` PR to clear seems like the way to go.

Reply:
I forgot to mention here, rustwide 0.15.1 was released with the updated `nix`. If you `cargo update -p rustwide -p nix -p bitflags` it should update just those crates and hopefully get the build passing.

Reply:
Thanks a lot for the heads-up! I upgraded both `crates-index-diff` and `rustwide` to their latest version, let's hope for the best.